cellular_automata/README.md

```diff
@@ -1,4 +1,8 @@
 # Cellular Automata
 
-* https://en.wikipedia.org/wiki/Cellular_automaton
-* https://mathworld.wolfram.com/ElementaryCellularAutomaton.html
+Cellular automata are a way to simulate the behavior of "life", no matter if it is a robot or cells.
```

Comment:
For me they're just shapes (or randomized generation of shapes).

Reply:
It's not random because (unless random play a part) two simulations with the same configuration and the same rules will lead to the same result
Cellular automaton were indeed created to replicate the behavior of "life": [https://en.wikipedia.org/wiki/Cellular_automaton#History](https://en.wikipedia.org/wiki/Cellular_automaton#History)

Reply:
```suggestion
Cellular automata are a way to simulate the behavior of "life", no matter if it is a robot or cell.
```